catalog/spatial/geocoding/spatial-geocoding-feature/src/main/java/org/codice/ddf/spatial/geocoding/feature/CatalogFeatureIndexer.java

```diff
@@ -143,12 +143,26 @@ private Metacard createMetacardForFeature(SimpleFeature feature) throws FeatureI
           new AttributeImpl(
               GeoEntryAttributes.GAZETTEER_SORT_VALUE,
               GeoCodingConstants.COUNTRY_GAZETTEER_SORT_VALUE));
+
+      if (isCountry(feature)) {
+        metacard.setAttribute(
+            new AttributeImpl(
+                GeoEntryAttributes.FEATURE_CODE_ATTRIBUTE_NAME,
+                GeoCodingConstants.POLITICAL_ENTITY));
+      }
+
       return metacard;
     }
 
     throw new FeatureIndexingException("Malformed feature");
   }
 
+  private boolean isCountry(SimpleFeature feature) {
+    return feature
+        .getAttribute(GeoCodingConstants.GEOMETRY_TYPE)
```

Comment:
✏️ If `feature.getAttribute(GeoCodingConstants.GEOMETRY_TYPE)` does not exist on `SimpleFeature` this will throw an NPE right?

http://docs.geotools.org/stable/javadocs/org/opengis/feature/simple/SimpleFeature.html#getAttribute-java.lang.String-

Could be refactored to...

```
private boolean isCountry(SimpleFeature feature) {
    Object geoType = feature.getAttribute(GeoCodingConstants.GEOMETRY_TYPE)
    return GeoCodingConstants.COUNTRY_TAG).equals(geoType);
}
```